src/popup/pages/Dashboard.vue

```diff
@@ -121,12 +121,16 @@ export default {
   data: () => ({
     buyBackground,
     chainNameBackground,
-    SIMPLEX_URL,
   }),
   computed: {
     ...mapState(['backedUpSeed', 'transactions']),
-    ...mapGetters(['getAccountPendingTransactions']),
+    ...mapGetters(['getAccountPendingTransactions', 'account']),
     ...mapState('accounts', ['activeIdx']),
+    getSimplexLink() {
```

Comment:
Computed is not a function but a property, so it shouldn't be named `getSimplexLink` but `simplexLink`